src/content/overview/backstage-portal/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Backstage portal
+description: Backstage is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so that engineers using your platform find all the information they need in the right place.
+weight: 30
+menu:
+  principal:
+    parent: overview
+    identifier: overview-backstage-portal
+last_review_date: 2024-07-01
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+---
+
+# Backstage portal
+
+[Backstage](https://backstage.io/) is your engineer's front end to the platform. We provide our self-service user interface as plugins for Backstage, so that engineers using your platform find all the information they need in the right place.
```

Comment:
I would like to link to the CNCF page for it to highlight our commitment to the initiative. I do the same with others.
https://www.cncf.io/projects/backstage/